olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/PrefetchTilesRequest.h

```diff
@@ -149,6 +149,35 @@ class DATASERVICE_READ_API PrefetchTilesRequest final {
     return *this;
   }
 
+  /**
+   * @brief Changes the prefetch behavior when prefetching a list of tiles. In
+   * case the tile does not exists, prefetch algorithm searches for the nearest
+   * parent and prefetch it.
+   *
+   * @param data_aggregation_enabled The boolean parameter that enables or
+   * disables the aggregation.
+   *
+   * @note Experimental. API may change.
+   * 
+   * @return A reference to the updated `PrefetchTilesRequest` instance.
+   */
+  inline PrefetchTilesRequest& WithDataAggregationEnabled(
+      bool data_aggregation_enabled) {
+    data_aggregation_enabled_ = data_aggregation_enabled;
+    return *this;
+  }
+
+  /**
+   * @brief Gets the data aggregation flag.
+   *
+   * @note Experimental. API may change.
+   *
+   * @return The data aggregation flag as boolean.
```

Comment:
Can we put it in the following way?

```suggestion
   * @return The data aggregation flag as a boolean value.
```